homeassistant/components/kodi/manifest.json

```diff
@@ -2,11 +2,15 @@
   "domain": "kodi",
   "name": "Kodi",
   "documentation": "https://www.home-assistant.io/integrations/kodi",
-  "requirements": ["pykodi==0.2.0"],
+  "requirements": [
```

Comment:
Is breaking the lines a new hassfest thing?

Reply:
It's black that does it

Reply:
There was a new black release which formats a few things differently IIRC.